src/app/logic/processGame.js

```diff
@@ -0,0 +1,146 @@
+import Answers from '../components/Answers';
+import VisualImage from '../components/VisualImage';
+import { getQuestion, initGame, isGameInitialized } from './quiz';
+
+class Game {
+  getTimeLeft() {
```

Comment:
Did you consider using getters and setter? get timeLeft(){} ?

Reply:
Yes, and creating setters and getters leads to overload calls heap with countdown